static/app/views/replays/utils/useReplayEvent.tsx

```diff
@@ -160,52 +152,26 @@ function useReplayEvent({eventSlug, location, orgId}: Options): Result {
       rrwebEvents: undefined,
       mergedReplayEvent: undefined,
     });
-    try {
-      const [event, rrwebEvents, replayEvents] = await Promise.all([
-        fetchEvent(),
-        fetchRRWebEvents(),
-        fetchReplayEvents(),
-      ]);
-
-      const breadcrumbEntry = mergeBreadcrumbsEntries(replayEvents || []);
-
-      // Get a merged list of all spans from all replay events
-      const spans = replayEvents.flatMap(
-        replayEvent => replayEvent.entries.find(isReplayEventEntity).data
-      );
-
-      // Create a merged spans entry on the first replay event and fake the
-      // endTimestamp by using the timestamp of the final span
-      const mergedReplayEvent = {
-        ...replayEvents[0],
-        breakdowns: null,
-        entries: [{type: EntryType.SPANS, data: spans}],
-        // This is probably better than taking the end timestamp of the last `replayEvent`
-        endTimestamp: spans[spans.length - 1]?.timestamp,
-      };
-
-      setState({
-        ...state,
-        fetchError: undefined,
-        fetching: false,
-        event,
-        mergedReplayEvent,
-        replayEvents,
-        rrwebEvents,
-        breadcrumbEntry,
-      });
-    } catch (error) {
-      setState({
-        fetchError: error,
-        fetching: false,
-
-        breadcrumbEntry: undefined,
-        event: undefined,
-        replayEvents: undefined,
-        rrwebEvents: undefined,
-        mergedReplayEvent: undefined,
-      });
-    }
```

Comment:
I think we should keep the try/catch, as we'll need the `fetchError`, but we should capture the error in Sentry so it doesn't get swallowed up and lost.